overlord/assertstate/assertstate_test.go

```diff
@@ -4878,3 +4902,191 @@ func (s *assertMgrSuite) TestForgetValidationSetPreferEnforcedByModelHappy(c *C)
 	err := assertstate.ForgetValidationSet(s.state, s.dev1Acct.AccountID(), "foo")
 	c.Check(err, IsNil)
 }
+
+type fakeAssertionStore struct {
+	storetest.Store
+
+	assertion           func(*asserts.AssertionType, []string, *auth.UserState) (asserts.Assertion, error)
+	seqFormingAssertion func(*asserts.AssertionType, []string, int, *auth.UserState) (asserts.Assertion, error)
+}
+
+func (f *fakeAssertionStore) Assertion(assertType *asserts.AssertionType, key []string, userState *auth.UserState) (asserts.Assertion, error) {
+	return f.assertion(assertType, key, userState)
+}
+
+func (f *fakeAssertionStore) SeqFormingAssertion(assertType *asserts.AssertionType, sequenceKey []string, sequence int, user *auth.UserState) (asserts.Assertion, error) {
+	return f.seqFormingAssertion(assertType, sequenceKey, sequence, user)
+}
+
+func (s *assertMgrSuite) TestValidationSetsFromModelOnline(c *C) {
+	s.testValidationSetsFromModel(c, false)
```

Comment:
nitpick: have a `const offline = false` that is passed to the method instead of directly the literal